tests/Examples/Docs/BulkPage.cs

```diff
@@ -100,7 +99,19 @@ public void Line405()
 			{ ""update"" : {""_id"" : ""3"", ""_index"" : ""index1"", ""_source"" : true} }
 			{ ""doc"" : {""field"" : ""value""} }
 			{ ""update"" : {""_id"" : ""4"", ""_index"" : ""index1""} }
-			{ ""doc"" : {""field"" : ""value""}, ""_source"": true}");
+			{ ""doc"" : {""field"" : ""value""}, ""_source"": true}",
+				e =>
+				{
+					e.ApplyBodyLineChanges(lines =>
```

Comment:
Maybe a `ApplyNdBodyChanges()` that uses `ParseJObjects` to read the body, then manipulate the `JObject`?

https://github.com/elastic/elasticsearch-net/blob/fea5445c345c4f041ab5835bb0c2b02d38fac7c0/tests/Examples/ResponseExtensions.cs#L109-L117